docs/DeveloperGuide.md

```diff
@@ -589,7 +615,59 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
   Use case ends.
 
-**Use case: UC06 - Showing search terms**
+**Use case: UC07 - Filtering applicants by passed interviews**
+
+**MSS**
+
+1.  User requests to <u>list applicants (UC01)</u>.
+2.  RecruitIn displays a list of applicants.
+3.  User requests to find all applicants with interviews that have already passed.
+4.  RecruitIn displays the applicants with interviews that have already passed.
+
+    Use case ends.
+
+**Extensions**
+
+* 1a. User requests to <u>find applicant (UC06)</u>.
+
+  Use case resumes at step 2.
+
+* 2a. The list of applicants is empty.
+
+  Use case ends.
+
+* 3a. The command for filtering by passed interviews is invalid.
```

Comment:
Do you think that it would be better to standardise to the same format as the other use cases: "The input format is invalid"?